### ISSUE TITLE
Dynamic number of tiles

### ISSUE THREAD

**Issue by mirh** [darktrojan/newtabtools#344]
Basically normal firefox behavior until 56. 
Example: if your firefox window is like 300x300 pixel, something like only a tile or two fill fit. 

Increase windows size (say, 1400x900) and you get more and more rows and/or columns. 
Kind-ish related to #155 maybe

**Comment by darktrojan** [comment]
I agree this would be nice, but it would take an enormous rewrite of a lot of things to make it work. And the there'd be complaints that it doesn't work the way it does now.

**Comment by mirh** [comment]
Well, that's why you have settings I guess?

**Comment by darktrojan** [comment]
I also have bills to pay, and only 24 hours in a day.
